assets/airports/airportLoadList.json

```diff
@@ -266,6 +266,13 @@
         "premium": false,
         "disabled": true
     },
+    {
+        "icao": "koma",
+        "level": "easy",
+        "name": "Omaha Eppley",
```

Comment:
Could we add in "Airfield"? It looks a bit lonely
![image](https://user-images.githubusercontent.com/13125311/152983746-edf2c333-d6a4-4d14-9cb4-ee7ed8ca65dd.png)